app/views/home/index.html.haml

```diff
@@ -1,2 +1,82 @@
+      .w-full.h-min.relative.mx-auto.overflow-x-hidden.overflow-y-hidden
+        #slider.h-80.w-80.flex.lg:gap-8.md:gap-6.gap-14.justify-start.transition.ease-out.duration-700
+          .flex.flex-shrink-0.relative.w-full.sm:w-auto
+            = image_tag "favicon.png", size: "400", class:"object-cover object-center w-full"
+            .bg-gray-800.bg-opacity-30.absolute.w-full.h-full.p-6
+              .flex.h-full.items-end.pb-2.justify-center
+                %h3.text-xl.lg:text-2xl.font-semibold.leading-5.lg:leading-6.text-white Plant 1
+          .flex.flex-shrink-0.relative.w-full.sm:w-auto
+            = image_tag "favicon.png", size: "400", class:"object-cover object-center w-full"
+            .bg-gray-800.bg-opacity-30.absolute.w-full.h-full.p-6
+              .flex.h-full.items-end.pb-2.justify-center
+                %h3.text-xl.lg:text-2xl.font-semibold.leading-5.lg:leading-6.text-white Plant 2
+          .flex.flex-shrink-0.relative.w-full.sm:w-auto
+            = image_tag "favicon.png", size: "400", class:"object-cover object-center w-full"
+            .bg-gray-800.bg-opacity-30.absolute.w-full.h-full.p-6
+              .flex.h-full.items-end.pb-2.justify-center
+                %h3.text-xl-text-white.lg:text-2xl.font-semibold.leading-5.lg:leading-6.text-white Plant 3
+      %button#next.-mr-10.focus:outline-none.focus:bg-gray-400.focus:ring-2.focus:ring-offset-2.focus:ring-gray-400{"aria-label" => "slide forward"}
+        %svg{fill: "none", height: "20", viewbox: "0 0 8 14", width: "12", xmlns: "http://www.w3.org/2000/svg"}
+          %path{d: "M1 1L7 7L1 13", stroke: "currentColor", "stroke-linecap" => "round", "stroke-linejoin" => "round", "stroke-width" => "2"}
+
+  .lg:col-span-12
+    %h2.font-semibold.text-2xl.mb-4 Community
+    %ul.flex.flex-wrap.items-center.text-sm.font-medium.text-center.text-white.border-b-2.border-emerald-600
+      %li.mr-2
+        %a.inline-block.p-4.bg-emerald-700.active:bg-emerald-700{"aria-current" => "page", href: "#"} All
+      %li.mr-2
+        %a.inline-block.p-4.text-black.bg-emerald-300.hover:bg-emerald-400{href: "#"} Popular topics
+      %li.mr-2
+        %a.inline-block.p-4.text-black.bg-emerald-300.hover:bg-emerald-400{href: "#"} Top growers
+      %li.ml-auto
+        %button.text-black.p-3.flex.items-center.hover:bg-emerald-300
+          %svg.mr-2{fill: "#047857", height: "28", version: "1.1", viewbox: "0 0 26 28", width: "26", xmlns: "http://www.w3.org/2000/svg"}
+            %title search1
+            %path{d: "M18 13c0-3.859-3.141-7-7-7s-7 3.141-7 7 3.141 7 7 7 7-3.141 7-7zM26 26c0 1.094-0.906 2-2 2-0.531 0-1.047-0.219-1.406-0.594l-5.359-5.344c-1.828 1.266-4.016 1.937-6.234 1.937-6.078 0-11-4.922-11-11s4.922-11 11-11 11 4.922 11 11c0 2.219-0.672 4.406-1.937 6.234l5.359 5.359c0.359 0.359 0.578 0.875 0.578 1.406z"}
+          = link_to "Search for growers" , people_path , :class => "text-base"
```

Comment:
Do the same search here as the plants search in previous comments, look at how the search function was implemented on the plants index page and do the same to the users

Reply:
The gray background is intentional to see better the white title. 

The effect is better with a real image.